kite-data/kite-data-core/src/test/java/org/kitesdk/data/event/thrift/Value.java

```diff
@@ -0,0 +1,427 @@
+/**
+ * Autogenerated by Thrift Compiler (0.7.0)
```

Comment:
I think we should check in the `.thrift` file that describes this type and add the thrift maven plugin to build this. Or maybe we can't because the maven plugin was added in thrift-0.9.2?